src/main/resources/data/carpet/tags/blocks/piston_weak_sticky.json

```diff
@@ -0,0 +1,69 @@
+{
+  "replace": false,
+  "values": [
```

Comment:
Ah okay... you want them to be weak sticky + breakable when enabled?

Reply:
Yes, if you push a weak sticky block into a wall, this decides if it will break or push the wall away